lib/merge.js

```diff
@@ -135,9 +135,6 @@ function doMerge(tree, path, dontExpand) {
           }
           continue;
         }
-        if (!item.ids) {
-          continue;
-        }
```

Comment:
^ This dead code removal does make me a little nervous, since it was copied from the Erlang code and may cover a case we're not aware of. @daleharvey could you take a look?